src/v/storage/types.h

```diff
@@ -167,12 +167,6 @@ struct offset_stats {
 
     model::offset dirty_offset;
     model::term_id dirty_offset_term;
-    // Base offset of the first batch in the most recent term stored in log
-    //
-    // The last_term_base offset will be used by the followers to inform the
```

Comment:
Move this second half of the comment also to the function?

Reply:
As far as I can tell that comment doesn't reflect what we're using it for anymore. So I skipped including it.